app/src/main/java/com/github/campus_capture/bootcamp/storage/ZoneDatabase.java

```diff
@@ -8,7 +9,7 @@
 import com.github.campus_capture.bootcamp.storage.dao.ZoneDAO;
 import com.github.campus_capture.bootcamp.storage.entities.Zone;
 
-@Database(entities = {Zone.class}, version = 1)
+@Database(entities = {Zone.class}, version = 2, autoMigrations = {@AutoMigration(from = 1, to = 2)})
```

Comment:
Does the migration mess up any of the previous zone databases or does it just streamline the transition?

Reply:
No the migration should be automatic and invisible. But before shipping it would be nice to reset it to 1

Migrations are useful when changing the schema of the database (which I did, by having the zone ID being auto-incremented), without having to restart from a fresh DB